cpp/velox/tests/FunctionTest.cc

```diff
@@ -39,6 +39,10 @@ using namespace facebook::velox::test;
 namespace gluten {
 class FunctionTest : public ::testing::Test {
  protected:
+  static void SetUpTestCase() {
+    memory::MemoryManager::testingSetInstance({});
```

Comment:
Required since this upstream commit:
f391c023e Use shrink pools by arbitrator for spill test in hash join uni test (#8932)